### ISSUE TITLE
Reports Dashboard

### ISSUE THREAD

**Issue by munyagz** [thedevdojo/voyager#5396]
## Version information

 - Laravel: v8
 - Voyager: v#.#.#
 - PHP: 7+
 - Database: [type] [version] (e.g. MySQL 8.0)

## Description of problem

Requesting a feature to create reporting dashboard and reports pages.
Just like the widgets page we can have great reports about transactions, products and such.
Instead of google analytics graphs we can have like sales compared monthly with a bar graph, or pie charts of a products and sales, to show which product is moving.

## Proposed solution
1. When I land on the dashboard I can have more than the widgets, but be able to create graphs with comparison relations.
2. have a reporting page where I can see transactions and I can filter with a date-range or categories.
## Alternatives considered
I have seen it in QuickAdmin system where you are able to generate dashboards

## Additional context
None at the moment.

**Comment by MrCrayon** [comment]
I think the scope is way too generic and there is no man power to develop and mantain such feature.

@fletch3555 what do you think?

**Comment by fletch3555** [comment]
I agree. This isn't something we'll be able to build or support.  The use-case is simultaneously too specific that not everyone will need it, and too generic that we wouldn't know what to include.
